invenio_rdm_records/services/secret_links/service.py

```diff
@@ -0,0 +1,287 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
```

Comment:
You will need to keep the copyright header from the original file here (it's important because it assigns ownership over the source code, and in this specific case it was done by TU Wien :-))

```
# Copyright (C) 2020-2021 CERN.
# Copyright (C) 2020-2021 Northwestern University.
# Copyright (C) 2021 TU Wien.
```